ToDoApp/ToDoApp/SignUp/SignUpViewModel.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  SignUpViewModel.swift
+//  ToDoApp
+//
+//  Created by Trong Hieu Vuong on 2022-12-20.
+//
+
+import Foundation
+import FirebaseAuth
+enum SignUpErrors: Identifiable, Hashable {
```

Comment:
```suggestion
enum SignUpError: Identifiable, Hashable {
```
enum type name should not be plural because one value represents one error
```
let error: SugnUpError = .invalidEmail
```